modules/python/src/custom_nodes/google_genmedia/veo3_api.py

```diff
@@ -106,17 +106,21 @@ def generate_video_from_text(
         """
         if not prompt or not isinstance(prompt, str) or len(prompt.strip()) == 0:
             raise ValueError("Prompt cannot be empty for text-to-video generation.")
-        if duration_seconds != 8:
+        valid_durations = (4, 6, 8)
```

Comment:
Change valid_durations to VEO3_VALID_DURATIONS = (4, 6, 8) and move it to constants.py and import it in veo3_api.py